kalite/coachreports/api_resources.py

```diff
@@ -54,9 +60,9 @@ class PlaylistProgressResource(PlaylistParentResource):
     ex_pct_struggling = fields.IntegerField(attribute='ex_pct_struggling')
     ex_pct_incomplete = fields.IntegerField(attribute='ex_pct_incomplete')
     ex_status = fields.CharField(attribute='ex_status')
-    quiz_exists = fields.BooleanField(attribute='quiz_exists')
-    quiz_status = fields.CharField(attribute='quiz_status')
-    quiz_pct_score = fields.IntegerField(attribute='quiz_pct_score')
+    # quiz_exists = fields.BooleanField(attribute='quiz_exists')
+    # quiz_status = fields.CharField(attribute='quiz_status')
+    # quiz_pct_score = fields.IntegerField(attribute='quiz_pct_score')
```

Comment:
If we ever want to reinstate the quiz feature in the future. Which, if we are going to explore different mastery models, and use the 'quiz' as the basis for giving people mixed sets of materials they have previously practiced, it could be useful.